airbyte-integrations/connectors/source-help-scout/source_help_scout/source.py

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from airbyte_cdk.sources.declarative.yaml_declarative_source import YamlDeclarativeSource
+
+"""
+This file provides the necessary constructs to interpret a provided declarative YAML configuration file into
+source connector.
+
+WARNING: Do not modify this file.
+"""
+
+
+# Declarative Source
+class SourceHelpScout(YamlDeclarativeSource):
+    def __init__(self):
+        super().__init__(**{"path_to_yaml": "help_scout.yaml"})
```

Comment:
for when the other file is renamed
```suggestion
        super().__init__(**{"path_to_yaml": "manifest.yaml"})
```